client/webserver/site/src/html/wallets.tmpl

```diff
@@ -323,7 +323,7 @@
        <div class="fs16 px-4 text-center">[[[Authorize the transfer with your app password.]]]</div>
         <div class="input-group mt-2">
           <label for="vSendPw" class="form-label m-2">[[[Password]]]: </label>
-          <input type="password" class="form-control select" id="vSendPw" autocomplete="off">
+          <input type="password" class="form-control select" id="vSendPw" autocomplete="current-password">
```

Comment:
I noticed there are several places where `autocomplete="off"` is used instead of `autocomplete="current-password"`, if it isn't on-purpose - I assumed it probably needs changing.

Reply:
https://github.com/decred/dcrdex/issues/414

Allowing autocomplete also tells the browser to remember what you entered.  We don't want that.

https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion#the_autocomplete_attribute_and_login_fields

> 
> Setting autocomplete="off" on fields has two effects:
> 
>    * It tells the browser **not to save data inputted by the user** for later autocompletion on similar forms, though heuristics for complying vary by browser.
>    * It **stops the browser from caching form data in the session history**. When form data is cached in session history, the information filled in by the user is shown in the case where the user has submitted the form and clicked the Back button to go back to the original form page.

Reply:
Okay, then I suppose instead better replace `autocomplete=current-password` and `autocomplete="new-password"` with `autocomplete="off"` on all views ? Since it is present on some in `master` which is why I wanted to change it in the first place (to make it uniform everywhere).

Just to mention, using password-autocomplete also allows for seamless integration with password manager (such as LastPass). But I suppose when dealing with money reducing attack surface is preferable.

Reply:
lastpass can and does ignore it in my experience.  It's quite aggressive... an irritating about it actually

Reply:
so, about other places on `master`, should I `autocomplete="off"` them as well ?